DP3TApp/Logic/Tracing/Reporting/FakePublishManager.swift

```diff
@@ -49,17 +49,17 @@ class FakePublishManager {
         return randomDay * secondsInADay
     }
 
-    private func getNewScheduleDate() -> Date {
-        Date(timeInterval: syncInterval(), since: now)
+    func getNewScheduleDate(oldDate: Date) -> Date {
+        Date(timeInterval: syncInterval(), since: oldDate)
     }
 
     @discardableResult
     func rescheduleFakeRequest(force: Bool = false) -> Date {
         queue.sync {
-            var nextDate = nextScheduledFakeRequestDateStore ?? getNewScheduleDate()
+            var nextDate = nextScheduledFakeRequestDateStore ?? getNewScheduleDate(oldDate: now)
 
             if nextDate <= now || force {
-                nextDate = getNewScheduleDate()
+                nextDate = getNewScheduleDate(oldDate: nextDate)
```

Comment:
should not now be used in this case as oldDate and the actual old case in the case above? because getNewScheduleDate(oldDate: now) will never be in the past? or is now not now?